pegasus/sites.v3/hourofcode.com/views/hoc_events_map_replacement.haml

```diff
@@ -0,0 +1,75 @@
+%link{rel: "stylesheet", type: "text/css", href: "/css/map.css"}
+
+%div{id: 'mapbox-map'}
+  %div{id: 'events-legend', class: 'mapbox-legend'}
+    .legend-title= hoc_s(:map_legend_title)
+    .color.brown
+    %div= hoc_s(:map_legend_hoc_events)
+    .color.teal
+    %div= hoc_s(:map_legend_cs_tech_jam)
+
+- mapbox_access_token = CDO.mapbox_access_token
+:javascript
```

Comment:
It's strongly preferred to get this JS into its own .js file in apps so that we get yarn building, linter coverage etc.

Reply:
Will do! I was following the (very old) format from the current map.

Reply:
I've left one line of JS in this haml file. It sets the access token for Mapbox, which is _much_ easier to do in a haml file as the access token will be stored as a secret.